site/src/server.rs

```diff
@@ -277,6 +277,8 @@ impl Server {
         // Refresh the landing page
         ctxt.landing_page.store(Arc::new(None));
 
+        let _ = ctxt.get_master_commits();
```

Comment:
I don't think this actually makes sense: it basically cannot help us get less stale data, since if things are cached they will still be cached despite this call. If the commits aren't already cached (i.e., more than 60 seconds since the previous call), then this would mean that we may refresh too early -- before triagebot updates the master commit list -- meaning that we have *delayed* getting the new commits for a full minute.

So I think we should just delete this line.